server/models/Collective.js

```diff
@@ -26,6 +26,9 @@ import roles from '../constants/roles';
 import activities from '../constants/activities';
 import { HOST_FEE_PERCENT } from '../constants/transactions';
 import { types } from '../constants/collectives';
+import status from '../constants/expense_status';
+
+const { PENDING, APPROVED, PAID } = status;
```

Comment:
It can be confusing to have this in the global scope for `Collective` as a collective can be pending for approval or approved by the host and the code will not reflect that we're talking about expenses when using `PENDING` below.

```es6
import ExpenseStatus from '../constants/expense_status';

if (... === ExpenseStatus.PENDING) {
  ... 
}
```

Would make more sense to me

Reply:
Oh, good point, good to know. Fixed.